src/caret_analyze/plot/graphviz/chain_latency.py

```diff
@@ -99,21 +99,46 @@ def get_attr_node(
     lstrip_s: float,
     rstrip_s: float
 ) -> GraphAttr:
+    def calc_latency_from_path_df(target_columns: list[str]) -> np.ndarray:
```

Comment:
The `calc_latency_from_path_df()` function does not appear to be called from the outside.
```suggestion
    def _calc_latency_from_path_df(target_columns: list[str]) -> np.ndarray:
```